src/Azs.ContainerRegistry.Admin/examples/New-AzsContainerRegistryQuota.md

```diff
@@ -0,0 +1,15 @@
+### Example 1: New Azs ContainerRegistry Quota
+```powershell
+PS C:\> New-AzsContainerRegistryQuota -QuotaName testquota -CapacityPerRegistryInGib 20 -NumberOfRegistry 20
```

Comment:
rename parameter to CapacityPerRegistryInGB

Reply:
We are following the same pattern as Azure do measuring in GiB instead of Gb